examples/lighting-app/nrfconnect/CMakeLists.txt

```diff
@@ -85,12 +85,16 @@ pw_proto_library(pigweed_lighting_protolib
 
 target_sources(app PRIVATE
   main/Rpc.cpp
+  ../../common/pigweed/RpcService.cpp
   ${NRFCONNECT_COMMON}/util/PigweedLogger.cpp
 )
 
 target_include_directories(app PRIVATE
   ${ZEPHYR_BASE}/subsys // required to get access to `log_output_std.h`
-)
+  ${PIGWEED_ROOT}/pw_sys_io/public
+  ${CHIP_ROOT}/src/lib/support
```

Comment:
Both `${CHIP_ROOT}/src/` and `${CHIP_ROOT}/src/lib` are already added to the include dirs. Could you refer to system and support headers using `#include <system/...` and `#include <support/...`, respectively? Having too many include directories has led to header file name collisions in the past.

Reply:
Rest of CHIP code (e.g. under src/ ) omits the 'lib' and 'support' and 'system' prefixes. I wanted to stay consistent with that even at the price of having to add more include paths (hopefully can be eliminated one day by linked libraries carrying their include paths?)

Reply:
Ah, ok. Let's leave it here then.